spring boot project

qf9ar8nv opened the pull request:
## PR 타입(하나 이상 선택해주세요.)

- [ ] 기능 추가
- [ ] 버그 수정
- [ ] 의존성, 환경 변수 업데이트
- [x] 기타

## 배경(Backgroud)

* Grabit 프로젝트에서 사용 할 backend project가 필요.
* start.spring.io 홈페이지의 spring starter를 통해 spring template 생성.


## 변경사항(Changes)

* Spring boot project 생성.
* Spring web dependency 추가.


## 결과(Result)

* main 함수 Run 실행하면 localhost:8080으로 실행 가능.
* RestController를 통해 API 호출 가능.

qf9ar8nv committed:
spring boot project

sally0226 commented:
![image](https://user-images.githubusercontent.com/43634786/147909240-c21ce0b7-c5a2-4aa7-a4a9-ce660cb91284.png)
위의 선택할 수 있는 옵션중에서 어떤 것들을 선택해 만들어진 프로젝트인지 명시해주시면 좋겠습니다!
Maven이랑 Gradle중에 뭘 골랐는지,
Spring Boot 버전은 몇인지 등등..

qf9ar8nv commented:
Project: Maven (Gradle도 편하고 많이 쓰이지만, Maven이 관련 자료가 많고 dependency 추가 할 때 공식 문서 참고가 쉬움)
Language: Java (Kotlin, groovy는 공부하지 않았고 자바로 
Spring boot: 2.6.2 (SNAPSHOT은 기능 안전화가 되지 않은 버전이므로, 제외하고 가장 최신 버전을 선택)
Packaging: Jar (WAR는 톰캣서버를 따로 구성해야하는 번거로움이 있어서 제외)
Java: 11 (가끔 너무 최신 자바에서는 지원하지 않는 라이브러리 등 경우도 있어서 11로 채택)
이렇게 선택했습니다.

sally0226 committed:
spring boot project 재설정

- maven -> gradle로 변경
- Java
- Spring Boot : 2.6.2
- Jar
- Dependencies : Spring Web
